### ISSUE TITLE
ESP8286 or ESP32?

### ISSUE THREAD

**Issue by stevemac00** [albahmed23/terrabloom_ec_fans#1]
Rpi is difficult to find these days so I was going to look into writing Lua code but wanted to check if you have anything not yet posted. Ideally I want to get this controlled via MQTT to work with homebridge/homekit automations.

**Comment by albahmed23** [comment]
Hi @stevemac00  - at the moment I do not have any LuaNode code for the ESP32 for this application. Does homebridge provide you with the MQTT broker? 
So essentially you are wanting a way to have your ESP32 subscribe to an MQTT topic that controls the fan speed?

**Comment by stevemac00** [comment]
Yes, that's it. Just thought I'd ask before starting. I use Mosquito for broker. For me the hard part is the wiring although I've not yet used PWM on an ESP. Appreciate the voltage shifter circuit.
